### PR TITLE
Added support for nodejs

### PR DESCRIPTION
**Pull request by SlavaRa**
@elsassph please review and merge after #142

### CODE REVIEW

**Comment by SlavaRa** [comment]
@elsassph please check this and merge if all ok, next target `hl`

File: src/massive/munit/client/HTTPClient.hx [hunk 1]
```diff
@@ -291,7 +291,7 @@ class URLRequest
 		#if (js || neko || cpp || java || cs || python || php)
 		client.onData = onData;
 		client.onError = onError;
-			#if js
+			#if(js && !nodejs)
```
Comment (elsassph) [inline src/massive/munit/client/HTTPClient.hx]: Why adding `-D nodejs` when you have `hxnodejs` flag already?

Comment (SlavaRa) [reply src/massive/munit/client/HTTPClient.hx]: -lib hxnodejs

:) but you can use `-lib nodejs` and others

File: tool/template/help_run.txt [hunk 2]
```diff
@@ -16,8 +16,9 @@ Optional Parameters:
 
    -[platform]       : One or more specific target platforms to compile
                        Valid options are '-swf', '-as3', '-js', '-neko',
-                       '-cpp', '-java', '-cs', '-python' and 'php'. If not target(s) provided 
+                       '-cpp', '-java', '-cs', '-python' and '-php'. If not target(s) provided
```
Comment (elsassph) [inline tool/template/help_run.txt]: `If not target(s) provided` with the text on the next line

File: tool/template/help_run.txt [hunk 3]
```diff
@@ -16,8 +16,9 @@ Optional Parameters:
 
    -[platform]       : One or more specific target platforms to compile
                        Valid options are '-swf', '-as3', '-js', '-neko',
-                       '-cpp', '-java', '-cs', '-python' and 'php'. If not target(s) provided 
+                       '-cpp', '-java', '-cs', '-python' and '-php'. If not target(s) provided 
                        all found test.hxml will be run.
+                       To run js tests with NodeJS, the compiler flags should include '-lib hxnodejs' or '-D hxnodejs'.
```
Comment (elsassph) [inline tool/template/help_run.txt]: Does it fit on screen?

**Comment by SlavaRa** [comment]
fixed

**Comment by SlavaRa** [comment]
@elsassph Thanks :)

**Pull request merged**